db/structure.sql

```diff
@@ -98,6 +98,18 @@ CREATE SEQUENCE public.agency_members_id_seq
 ALTER SEQUENCE public.agency_members_id_seq OWNED BY public.agency_members.id;
 
 
+--
+-- Name: ar_internal_metadata; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.ar_internal_metadata (
+    key character varying NOT NULL,
+    value character varying,
+    created_at timestamp(6) without time zone NOT NULL,
+    updated_at timestamp(6) without time zone NOT NULL
+);
```

Comment:
I was looking up some details on this table because I don't have it locally. How did you create it?

Reply:
Honestly idk, it may have been created by the rails generator I used for the model / migration